src/main/drivers/dshot_bitbang.c

```diff
@@ -519,7 +521,7 @@ static bool bbUpdateStart(void)
                 dshotTelemetryState.motorState[motorIndex].telemetryValue = value;
                 dshotTelemetryState.motorState[motorIndex].telemetryActive = true;
                 if (motorIndex < 4) {
-                    DEBUG_SET(DEBUG_DSHOT_RPM_TELEMETRY, motorIndex, value);
+                    DEBUG_SET(DEBUG_DSHOT_RPM_TELEMETRY, motorIndex, calcEscRpm(value) / 10);
```

Comment:
This will be problematic, as `calcEscRpm()` is conditional to `USE_ESC_SENSOR`. One way to address this will be to move it into a different compilation unit, as its use is not exclusive to the ESC sensor functionality any more.
But the more basic question here is if the log should carry a 'refined' metric, or the most basic value available, and leave it to the consuming application to convert the metric into whatever the user wants to see. In general, there is no expectation whatsoever that blackbox logs should be 'human readable', as they are inherently not (unless one can read binary). So there will always be the need to use software to read these logs, and smarts to convert basic data streams into more refined ones can be added to this software.
A proposal to facilitate conversion from eRPM to RPM is already under way: https://github.com/betaflight/betaflight/pull/10894
Nonwithstanding of this, the observation that the current value range is problematic is a good one, and some form of scaling should be done here to make it possible to represent all possible 
 / reasonable values without clipping.

Reply:
A benefit of making this more human readable would be when using the Debug display on the Sensors tab. Apart from that I don’t suppose it makes much difference where the conversion is done. But ultimately RPM is better than eRPM in the final presentation.

Reply:
Even for the Debug display in the Sensors tab in configurator I would argue that if we wanted to introduce data conversion in order to make the graphs easier to read, the right way to do so would be to add the conversion in configurator, especially in cases where the converted data would otherwise require more debug options, or replaced a more basic metric that can serve as the basis for other derived values.
But in this case none of eRPM and RPM is 'more basic' than the other, and both can be converted into the other given the motor pole count, so it does not really matter.